circle.yml

```diff
@@ -8,11 +8,14 @@ dependencies:
     - ~/.RLibrary
     - ~/dart-sdk/bin
     - ~/.cabal
+    - ~/infer-linux64-v0.7.0
   pre:
     - echo 'export PATH=$PATH:~/coala-bears/node_modules/.bin' >> ~/.circlerc
     - echo 'export LINTR_COMMENT_BOT=false' >> ~/.circlerc
     - echo 'export PATH=$PATH:~/dart-sdk/bin' >> ~/.circlerc
     - echo 'export PATH=$PATH:~/.cabal/bin' >> ~/.circlerc
+    - echo 'export PATH=$PATH:~/infer-linux64-v0.7.0/infer/bin' >> ~/.circlerc
+    - echo 'alias python2.7=python' >> ~/.circlerc
```

Comment:
I am not sure what you want to do here, but if you want to use `python2.7`, you will have to set an exact local version explicitly.

Something like
````
pyenv install 2.7.10
pyenv local 2.7.10
```

https://github.com/yyuu/pyenv/issues/34